### ISSUE TITLE
Simplified export of png overlays compatible to the PLANET system on HALO

### ISSUE THREAD

**Issue by janhclem** [Open-MSS/MSS#1898]
png overlays without axis, titles etc. could be used in systems like PLANET as map overlays that are also accessible for operators on the aeroplane. I am looking for a simple workflow/option to export png overlays from the MSS UI to png overlays. An option "export as overlay" would be the easiest way in my opinion. The overlays should have a high enough resolution to allow zooming into them within PLANET a bit, but not too high because of performance. Furthermore, they need to be in cylindrical coordinates, without borders, axis and so on.

**Comment by ReimarBauer** [comment]
@janhclem Please verify if the files in the wms cache can be used
you can configure the dir to store them by the configuration editor,

![Screenshot_20230817_085242](https://github.com/Open-MSS/MSS/assets/278728/0af9bf6e-72de-419c-8c5c-54ddd855259d)

This is what the server delivers. 

Based on these files, what needs to be different?

**Comment by janhclem** [comment]
Thanks. The files work very well as overlays for PLANET. The resolution however should be increased because I want to zoom into them for a factor of around 10. Best thing would be if I could try a coupled of different resolutions. Additionally, it would be good to have png files with human readable names.

![6bf1a79fbef6df6c88cf6ad84e479a34](https://github.com/Open-MSS/MSS/assets/77122332/bde52618-f29b-4f9e-b685-ea3d52bc62d0)
![bf5fe55df16c4fe442fc9f5ed5da1b78](https://github.com/Open-MSS/MSS/assets/77122332/f334fb71-e867-4d52-b31e-a1076ed9c384)

**Comment by ReimarBauer** [comment]
@jgrooss I assume this needs a different resolution of the used data, or?

**Comment by ReimarBauer** [comment]
@janhclem the Cache is not meant as store but we maybe can add use the UI to 'save as' so you can store it under a human readable Format. Have also a look on the options. May be you can disable there anything already.

**Comment by janhclem** [comment]
> @jgrooss I assume this needs a different resolution of the used data, or? 

I don't think so and should have clarified that. A higher resolution of the figure would be better to see borders sharply, higher resolution of the CLaMS data is not needed. The borders are too blurry that is the main problem (see below), the CLaMS fields are okay.
![example](https://github.com/Open-MSS/MSS/assets/77122332/cee4f1f8-0f51-4886-adf8-b9c7b7af1adb)

**Comment by jgrooss** [comment]
The problem was, that for non-total hemispherical sections it did not seem to work in PLANET. The resolution of the plot is always defined by the msui screen resolution. for this case it may be an idea to get a higher resolution by requesting it from the server by an appropriate URL

**Comment by jgrooss** [comment]
Or one could leave out the shorelines here completely and make the overlay transparent in PLANET

**Comment by ReimarBauer** [comment]
We have these Layout sizes configurable.  Please give it a try, change it to a higher value.
![layout_sizes](https://github.com/Open-MSS/MSS/assets/278728/55fa6c18-5b72-45a8-9a61-21651042b1eb)

**Comment by ReimarBauer** [comment]
when this works, then likly mss autoplot is the solution @swsrkty what do you think about this. Fetching data without further addons by the "msui"?

**Comment by janhclem** [comment]
Varying the layout parameter for top view somehow does not affect the figures I find in the cache folder. The resolution only depends on the size if the window I have opened.

**Comment by ReimarBauer** [comment]
That's a good question ;)
https://github.com/Open-MSS/MSS/issues/1918

Please set immutable too

![immutable](https://github.com/Open-MSS/MSS/assets/278728/a2a6d9e8-96e6-472d-ae2f-8ac71ff2f2e4)

**Comment by ReimarBauer** [comment]
the immutable, blocks resizing of the view area, which is wanted for the case you need the exact sizes.
currently we define the view area by this. The fetched image is smaller. This gives a 916 × 458 pixels for the topview view area by 1200 x 1000

**Comment by janhclem** [comment]
Thanks, now I can scale the figures. With 9160x7020 it looks much better on PLANET. I think that is enough.

The side effect now is that my msui window is to large for normal usage. So I think the last step would be to make the export convenient or even automated.   You mentioned mss autopilot. I will have a look at it...

**Comment by swsrkty** [comment]
Hi @janhclem, please have a look at https://mss.readthedocs.io/en/stable/mssautoplot.html and let me know if I can help in anyway.

**Comment by ReimarBauer** [comment]
This should become documented as a use case.

**Comment by ReimarBauer** [comment]
That is with the current mssautoplot not possible. It does not lookup and use the layout definitions of topview, sideview or linearview.

**Comment by ReimarBauer** [comment]
When the layout option is included into mssautoplot, it needs another option to return only the image  without axis, titles etc. for this usecase
we add a --raw click command
